lib/common_test/doc/src/ct_suite.xml

```diff
@@ -596,25 +596,40 @@
 	<p>Elements from the <c>Config</c> list can, for example, be read
 	  with <c>proplists:get_value/2</c> in STDLIB.</p>
 
-	<p>If you decide not to run the test case after all, return
-	  <c>{skip, Reason}</c>. <c>Reason</c> is then
-	  printed in field <c>Comment</c> on the HTML result page.</p>
-
-	<p>To print some information in field <c>Comment</c> on the HTML
-	  result page, return <c>{comment, Comment}</c>.</p>
-
-	<p>If the function returns anything else, the test case is
-	  considered successful. The return value always gets printed
-	  in the test case log file.</p>
+	<p>Possible return values are:</p>
+	<taglist>
+	  <tag><c>{fail, Reason}</c></tag>
+	  <item>The test case is considered failed, and the <c>Reason</c>
+	    will be logged.</item>
+	  <tag><c>{skip, Reason}</c></tag>
+	  <item>The test case is considered skipped, and the <c>Reason</c>
+	    will be logged.</item>
+	  <tag><c>{comment, Comment}</c></tag>
+	  <item>The test case is considered successful, and the <c>Comment</c>
+	    will be logged.</item>
+	  <tag><c>{save_config, SaveConfig}</c></tag>
+	  <item>The test case is considered successful, and the <c>SaveConfig</c>
+	    will be stored in the <c>Config</c> (see section
+	    <seeguide marker="dependencies_chapter#save_config">Saving
+	    Configuration Data</seeguide> in the User's Guide).</item>
+	  <tag><c>{skip_and_save, Reason, SaveConfig}</c></tag>
+	  <item>The test case is considered skipped, the <c>Reason</c>
+	    will be logged, and the <c>SaveConfig</c> will be stored in the
+	    <c>Config</c> (see section
+            <seeguide marker="dependencies_chapter#save_config">Saving
+            Configuration Data</seeguide> in the User's Guide).</item>
+	</taglist>
+
+	<p>If the function returns any other term, the test case is considered
+	  successful. It is common practice to explicitly return <c>ok</c> from
+	  successful test cases.</p>
```

Comment:
I am not sure we want to spread that practice, even though it is quite used it can be very useful not to return ok as you
can then see in the log what was the result of the last action in the test case.

Reply:
Ok, agreed :+1:

Reply:
Removed the sentence in the last commit.